hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/GCMultipleMergedRegionsProcedure.java

```diff
@@ -78,6 +78,13 @@ protected Flow executeFromState(MasterProcedureEnv env, GCMergedRegionsState sta
       switch (state) {
         case GC_MERGED_REGIONS_PREPARE:
           // Nothing to do to prepare.
```

Comment:
Hi @nyl3532016, can you put your above comment "If GCMultipleMergedRegionsProcedure processing is slower than the CatalogJanitor's scan interval, it will end resubmitting GCMultipleMergedRegionsProcedure for the same region, we can skip duplicate GCMultipleMergedRegionsProcedure" to the code above? It will help other developers's code reading easier. After that, you can merge, thanks.

Reply:
ok,thanks, I know. I will add this comment.